ngraph/python/src/pyngraph/variant.hpp

```diff
@@ -40,15 +40,18 @@ extern void regclass_pyngraph_VariantWrapper(py::module m, std::string typestrin
 
     variant_wrapper.def(py::init<const VT&>());
 
-    variant_wrapper.def("__eq__",
-                        [](const ngraph::VariantWrapper<VT>& a,
-                           const ngraph::VariantWrapper<VT>& b) { return a.get() == b.get(); },
-                        py::is_operator());
-    variant_wrapper.def("__eq__",
-                        [](const ngraph::VariantWrapper<std::string>& a, const std::string& b) {
-                            return a.get() == b;
-                        },
-                        py::is_operator());
+    variant_wrapper.def(
+        "__eq__",
+        [](const ngraph::VariantWrapper<VT>& a, const ngraph::VariantWrapper<VT>& b) {
+            return a.get() == b.get();
+        },
```

Comment:
the old one was better IMO

Reply:
I didn't find style configuration flag to align this with the old style.